src/reconcile/rec_visibility.c

```diff
@@ -160,7 +163,14 @@ __rec_append_orig_value(
 
             WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, &size));
             total_size += size;
-            tombstone->txnid = unpack->tw.stop_txn;
+            /*
+             * When evicting in recovery, we need to clear the transaction id as it is written
```

Comment:
Can you elaborate more on what you mean by "evicting in recovery"?

Reply:
eviction can happen during rts with the WT_CONN_RECOVERY flag set.